pkg/build/node/const.go

```diff
@@ -217,10 +218,65 @@ spec:
         securityContext:
           privileged: false
           capabilities:
-             add: ["NET_ADMIN"]
+            add: ["NET_RAW", "NET_ADMIN"]
       volumes:
         - name: cni-cfg
           hostPath:
             path: /etc/cni/net.d
 ---
+# ipmasq agent
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ip-masq-agent
+  namespace: kube-system
+  labels:
+    tier: node
+    app: kindnet
+data:
+  config: |-
+    nonMasqueradeCIDRs:
+      - 10.244.0.0/16
+    masqLinkLocal: false
```

Comment:
This value is available to kind and in the kubadm config map, I guess that we should be able to pass it in

Reply:
yep, have some ideas for that

Reply:
this is now templated.